wsus/README.md

```diff
@@ -0,0 +1,15 @@
+# Set up WSUS by using the automation script
+
+This content supports the [Plan deployment for updating Windows VMs in Azure](https://github.com/MicrosoftDocs/architecture-center/tree/master/docs/example-scenario/wsus) Azure Architecture Center article. For information about these files please see that article.
```

Comment:
Can you please link to the live docs page instead of the the github repo's version of this page?

Reply:
Done